lib/wanda/result/execution_result.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Wanda.Result.ExecutionResult do
```

Comment:
I would find other place to store this schemas. Somewhere within `execution` (as it belongs to this domain) or entities folder. Maybe `Wanda.Execution.Entities.ExecutionResult`

Reply:
Thanks Xabi, this is a good point.

We could indeed have an `wanda/execution/entities` directory where we could place this `execution_result.ex` entity.
Also the proposed namespacing `Wanda.Execution.Entities.ExecutionResult` sounds quite ok to me.

I'd love to have other inputs on this, though. @trento-project/developers what do you think?

Reply:
We could call the folder schema since entity is ambiguous and it is not really used anywhere in the code. We could also add the context functionality to the `wanda/execution.ex `. which already has start execution receive facts and it will also have the responsibility of retrieving results.

Reply:
I think that having `result/something.ex` is idiomatic and super elixirish, about the name of the file I really don't know, no hard feeling

Reply:
Alright, before this becomes an endless thread, let's stick with the standard provided by the framework.

![image](https://user-images.githubusercontent.com/8167114/192840164-03e4f5cc-572f-4a71-aa94-334d597fe969.png)

So, going with:
`wanda/results` as the directory
`wanda/results/execution_result.ex` as the schema file
`wanda/results.ex` as the context/service

Let's see what kind of refactoring it needs in the future.

Reply:
I would go with @fabriziosestito suggestion in any case `wanda/execution/execution_result_schema.ex`. I still see that this logic should go in the `execution` folder, as it belongs there. And yes, you can put the functions to query/save the entries in `execution.ex` if you like, it looks good to me.

Reply:
keeping that in mind for the following round! Thanks!